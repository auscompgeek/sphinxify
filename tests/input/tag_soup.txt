be <b>bold</b>
be <strong>strong</strong>
<i>italics</i>
<em>emphasis</em>
<sub>subscript</sub>
<sup>superscript</sup>

<strong>multiline
  strong</strong>
<i>multiline
  italics</i>
