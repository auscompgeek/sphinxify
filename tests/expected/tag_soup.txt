be **bold**
be **strong**
*italics*
*emphasis*
:sub:`subscript`
:sup:`superscript`

:strong:`multiline
strong`
:emphasis:`multiline
italics`
